WcaOnRails/app/javascript/auto-numeric/index.js

```diff
@@ -0,0 +1,67 @@
+import AutoNumeric from 'autonumeric';
+import currenciesData from 'wca/currenciesData.js.erb';
+
+let autoNumericObjectMap = new Map();
```

Comment:
Just as a quick note, I think this is a potential memory leak: we only ever add things to this map, we never remove them. Honestly, I think that's ok, though, since everything gets cleared out when we navigate to a new page.